apps/api/migrations/secure-to-boolean/secure-to-boolean-migration.ts

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable no-console */
+import '../../src/config';
+import { IntegrationRepository } from '@novu/dal';
+import { NestFactory } from '@nestjs/core';
+import { AppModule } from '../../src/app.module';
+
+const integrationRepository = new IntegrationRepository();
+
+export async function run() {
+  await NestFactory.create(AppModule, {
+    logger: false,
+  });
+
+  console.log('Start migration - update credentials.secure from string to boolean');
+
+  console.log('Updating from "true" to true...');
+  const resultTrue = await updateTrueValues();
+  console.log(`Matched: ${resultTrue.matchedCount}\nModified: ${resultTrue.modifiedCount} \n`);
+
+  console.log('Updating from "false" to false...');
+  const resultFalse = await updateFalseValues();
+  console.log(`Matched: ${resultFalse.matchedCount}\nModified: ${resultFalse.modifiedCount}`);
+
+  console.log('End migration.');
```

Comment:
Use `Logger` from `@nestjs/common` rather than `console.log` so we can track in out telemetry tools.

Reply:
sure. In other migration scripts I have seen console.logs so I decided to use them in my script as well.

I switched to Nest.js Logger in recent changes.